secrets/rules/rule_test.go

```diff
@@ -0,0 +1,28 @@
+package rules_test
+
+import (
+	"testing"
+
+	"github.com/checkmarx/2ms/secrets/rules"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func Test2msRules(t *testing.T) {
+	t.Parallel()
+
+	testsRules := []struct {
+		name     string
+		validate func() *config.Rule
+	}{
+		{name: "AuthenticatedURL", validate: rules.AuthenticatedURL},
+	}
```

Comment:
We will need to add each added rule manually to here, but I'm not worried, because the `validation` is running every time we're loading a rule, and since we're loading all the rules in the `loadAllRules` function, it will always be tested.

Reply:
I agree, looks good 👍